### PR TITLE
Update monorepo

### PR DESCRIPTION
**Pull request by i1i1**

### CODE REVIEW

**Review by ozgunozerk — DISMISSED** [review]

File: src/farmer.rs [hunk 1]
```diff
@@ -344,7 +335,7 @@ pub struct Farmer {
     cmd_sender: Arc<Mutex<Option<oneshot::Sender<oneshot::Sender<()>>>>>,
     reward_address: PublicKey,
     plot_info: Arc<HashMap<PathBuf, Plot>>,
-    node: Node,
+    _node: Node,
```
Comment (ozgunozerk) [inline src/farmer.rs:338]: why are we even returning `node` inside the `Farmer` struct?
and also, if it is necessary, why is it underscored?

Comment (i1i1) [reply src/farmer.rs:338]: I can remove it at all, but I believe we can use it in future.

**Comment by i1i1** [comment]
IDK why, but tests are failing, while `sync` example is working. Will spend some time on investigation

**Comment by ozgunozerk** [comment]
Please let me know when this merges.
Current version of SDK is using a revision of subspace monorepo, which does not exist (probably you used that rev while the target commit was a not a merged branch, and now deleted). This is causing the `release` build of cli to fail. I'll now reference this branch in the cli for fixing the issue temporarily, but I should update the revision for SDK once again when this merges.

**Comment by i1i1** [comment]
Closed in favor of #65

**Pull request closed without merging**